changelog.md

```diff
@@ -1,41 +1,39 @@
 * Fixed issue that led to `DeprecationWarning` in the kymotracker widget.
 * Fixed error in kymotracking documentation. The tutorial previously indicated the incorrect number of samples for `sample_from_image`.
-
-#### Bug fixes
-
+* Fixed bug that would implicitly convert `Kymograph` and `Scan` `timestamps` to floating point values. Converting them to floating point values leads to a loss of precision. For more information see [files and channels](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/file.html#channels) for more information.
 * Fixed bug where color-aligned data was returned from `TiffFrame.data` although alignment was not requested (e.g., `CorrelatedStack("filename.tiff", align=False)`). This bug was introduced in `v0.10.1`.
 
 #### Deprecations
 
-* `CorrelatedStack.raw` has been deprecated and will be removed in a future release.
+* `CorrelatedStack.raw` has been deprecated and will be removed in a future release. Use `CorrelatedStack.get_image()` instead.
 
 #### Breaking changes
 
-* Changed default for `viscosity` in force calibration models. When omitted, `pylake` will use the viscosity of water calculated from the temperature. Note that this results in the default (when no viscosity or temperature is set) changing from `1.002e-3`  to `1.00157e-3`. See: [force calibration](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/force_calibration.html).
-* Units are now included in the headers for exported kymograph traces. The header now reads:</br>
-`# line index;time (pixels);coordinate (pixels);time (seconds);position ({unit});counts (summed over {n} pixels)`</br>
-where `{unit}` is either `um` or `kbp` depending on the calibration of the kymograph.
-* Added more input validation for force calibration. We now force bead and sample density to be more than 100 kg/m^3 when specified. Temperature should stay between 5 and 90 degrees Celsius and viscosity should be bigger than 0.0003 (viscosity of water at 90 degrees Celsius).
+* Changed default for `viscosity` in force calibration models. When omitted, `pylake` will use the viscosity of water calculated from the temperature. Note that this results in the default (when no viscosity or temperature is set) changing from `1.002e-3`  to `1.00157e-3 Pa*s`. Please refer to [force calibration](https://lumicks-pylake.readthedocs.io/en/latest/tutorial/force_calibration.html) for more information.
+* Units are now included in the headers for exported kymograph traces. The header now includes the appropriate units for each data column (either `um` or `kbp` depending on the calibration of the kymograph). Any code that hardcoded the header names directly should be updated.
```

Comment:
The first two sentences repeat each other. Maybe something like this. Do also mention the API that does this exporting. It would be clearer.
```suggestion
* Units are now included in the headers for exported kymograph traces (either `um` or `kbp` depending on the calibration of the kymograph). Any code that hardcoded the header names directly should be updated.
```

Reply:
Good point! Changed it to the following:

* Units are now included in the headers for kymograph traces exported from the widget or with `KymoLineGroup.save()` (either `um` or `kbp` depending on the calibration of the kymograph). Any code that hardcoded the header names directly should be updated.